src/classes/GS_ApplicationStatusControllerTest.cls

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
The format we follow in NPSP for test classes is to use `_TEST` at the end: `GS_ApplicationStatusController_TEST`.